doc/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+### Description
+
+Write in as much detail as possible on the problem that this PR solves or the feature that it adds.
+This will give the reviewers more context on this pull request.
+
+### Screenshots
+
+Attach screenshots or diagrams, if applicable.
+
+### Developer checklist
+- [ ] If there is a Jira task, c/p the task description to this PR
+- [ ] Assign yourself to this pull request
+- [ ] Open a DRAFT pull request if it's not ready for review yet
+- [ ] Tag the reviewers for review
+- [ ] Attach appropriate labels to the PR (`enhancement`, `bug`, `work-in-progress`, etc.)
```

Comment:
I think we do not need `work-in-progress` as we use draft prs

Reply:
We can make it optional, so that the PRs are still filterable by that label, because filtering draft PRs is currently non-intuitive (you need to add `is:draft` to the search query.